src/main/java/com/uid2/admin/store/writer/EncryptionKeyStoreWriter.java

```diff
@@ -32,7 +33,7 @@ public void upload(Collection<EncryptionKey> data, JsonObject extraMeta) throws
             json.put("created", key.getCreated().getEpochSecond());
             json.put("activates", key.getActivates().getEpochSecond());
             json.put("expires", key.getExpires().getEpochSecond());
-            json.put("secret", key.getKeyBytes());
+            json.put("secret", Base64.getEncoder().encodeToString(key.getKeyBytes()));
```

Comment:
Who reads this data? I assume this will change what is written.

Reply:
The problem here is that the Base64 values actually get incorrectly encoded when writing to the site-specific files

One example (global file above, site-specific file below):
RgyxkP4yP1gYhCINq7O9PxM6jX+etPqSQluZxjB1aG8=
RgyxkP4yP1gYhCINq7O9PxM6jX-etPqSQluZxjB1aG8

The `+` became a `-`, and this causes an error when trying to decode the base64 string

Reply:
This `upload()` function is called mainly when writing files to S3

Reply:
If that is the case, then the readers of the S3 files will be expecting something different. I think this is very risky.

Reply:
Yeah I think this might actually be an existing issue and will appear for special characters like + or /

Whatever is stored in `keyBytes` array is correct. The only issue here is converting the byte array to the actual base64 string

Reply:
So adding the `encodeToString()` function above should rectify the aforementioned inconsistency

Reply:
as mentioned in uid2-operator, we actually need all the Base64 encodes for `JsonObject.put()` with `byte[]` data types because of the Vert.x 4 deprecation of their `JsonObject` implementation

Vert.x 3 encodes the byte array as a base64-encoded string
```
    public JsonObject put(String key, byte[] value) {
        Objects.requireNonNull(key);
        this.map.put(key, value == null ? null : Base64.getEncoder().encodeToString(value));
        return this;
    }
```

Vert.x 4 keeps the byte array as is
```
  public JsonObject put(String key, Object value) {
    Objects.requireNonNull(key);
    map.put(key, value);
    return this;
  }
```